pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -145,7 +145,7 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 	}
 
 	releaseImage := defaultReleaseImage
-	if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"); ok && ri != "" {
+	if ri, ok := os.LookupEnv("_OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"); ok && ri != "" {
```

Comment:
why?

Reply:
So that the CI doesn't override our image with the latest. It's a hack, but we don't support overriding the release image, so nobody should need this.

Reply:
Notes on this in the commit message too, but since we've used this pattern a few times now I didn't copy that paragraph into the PR post.